src/App.tsx

```diff
@@ -15,36 +15,85 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+export enum SortKeys {
+  ALPHABETICAL = 'Alphabetical',
+  LENGTH = 'Length',
```

Comment:
```suggestion
export enum SortType {
  Alphabetical = 'alphabetical',
  Length = 'length',
```
Enum properties shouldn't be in upper case, check the example from the [doc](https://mate-academy.github.io/fe-program/js/extra/typescript)
![image](https://github.com/user-attachments/assets/bf97c961-310b-4da5-a245-ebaf6e88d099)